resources/authgear/templates/en/web/settingsv2/__wide_page_frame.html

```diff
@@ -0,0 +1,37 @@
+{{ define "settingsv2/__wide_page_frame.html" }}
```

Comment:
There is no `settingsv2`.

Reply:
@louischan-oursky You prefer put it under `authflowv2/`?

Reply:
Yes.

Reply:
@yyorkchan Please move all stuff under `authflowv2` Thanks!